strr-examiner-web/app/components/Host/SupportingInfo.vue

```diff
@@ -76,6 +82,26 @@ const hasBlSection = computed((): boolean =>
   businessLicenseExpiryDate
 )
 
+const needsBusinessLicenseDocumentUpload = computed(() => {
+  if (!isBusinessLicenseDocumentUploadEnabled.value || !activeReg.value) {
+    return false
+  }
+  const jurisdiction = activeReg.value.unitDetails?.jurisdiction
+  const needsBusinessLicense = activeReg.value.status === RegistrationStatus.ACTIVE &&
+    needsBusinessLicenseUpload(
+      jurisdiction,
+      isBusinessLicenseDocumentUploadEnabled.value
```

Comment:
You don't need to pass `isBusinessLicenseDocumentUploadEnabled.value` because it is taken care of in the condition on line 86.